Launching-new-features.md

```diff
@@ -186,88 +184,22 @@ spyOnProperty(platformFeatureService, 'status', 'get').and.returnValue(
 
 #### Backend
 
-For example, if you wish to write a test with the required feature flag enabled, you can do so by following the steps below (the approach to testing with the feature flag disabled is the same up to step 4):
+For example, if you wish to write a test with the required feature flag enabled, you can do so by following the steps below:
```

Comment:
Replace with:

In general, you should test both the cases in which the feature flag is enabled and when it is not. To write a test with the feature flag enabled, you can do the following:

(Also you might want to explain somewhere how to write the disabled case too.)

Reply:
Done